custom_components/frigate/__init__.py

```diff
@@ -170,8 +174,13 @@ async def async_setup(hass: HomeAssistant, config: Config) -> bool:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up this integration using UI."""
+    info = await system_info.async_get_system_info(hass)
 
-    client = FrigateApiClient(entry.data.get(CONF_URL), async_get_clientsession(hass))
+    client = FrigateApiClient(
+        entry.data.get(CONF_URL),
+        info.get("timezone", "utc"),
```

Comment:
Nit: Should "UTC" be capitalized?

Reply:
I believe for pytz (what we are using on the backend) it is lower case only